src/__tests__/components/BasketballCourt/helper.tsx

```diff
@@ -0,0 +1,36 @@
+import Konva from "konva";
+import React from "react";
+import ReactDOM from "react-dom";
+
+type Component = React.ReactNode;
+interface AppProps {
+  onUpdate: (prop: unknown) => void;
+  children: Component;
+}
+
+const render = async (component: Component) => {
```

Comment:
`import {ReactNode} from 'react`
`component: ReactNode`

seems you don't need to reassign a type name to it